### ISSUE TITLE
Cyclical source disconnection

### ISSUE THREAD

**Issue by ilfuriano** [aler9/rtsp-simple-server#123]
## Which version are you using?

v0.12.1

## Which operating system are you using?

Windows / Linux

Description

The source disconnects form server after few seconds and it seems a cyclical behavior.
I tryed tinyCAM on Android and VLC on win 10 as client and the issue is the same.

If I use the source directly in the client, it works fine. The problem is the proxy server. 

My configuration is the default one, my path is:
``` yaml
....
paths:  
 cam1_low:
    source: rtsp://192.168.x.x:554/user=myuser&password=mypassword&channel=1&stream=1.sdp?real_stream
    sourceProtocol: tcp
    sourceOnDemand: yes
```
The log:
``` txt
2020/11/12 13:21:05 [0/0/0] rtsp-simple-server v0.12.1
2020/11/12 13:21:05 [0/0/0] [UDP/RTP server] opened on :8000
2020/11/12 13:21:05 [0/0/0] [UDP/RTCP server] opened on :8001
2020/11/12 13:21:05 [0/0/0] [TCP server] opened on :8554
2020/11/12 13:21:13 [1/0/0] [client 192.168.1.80:38346] connected
2020/11/12 13:21:13 [1/0/0] [client 192.168.1.80:38346] OPTIONS
2020/11/12 13:21:13 [2/0/0] [client 192.168.1.80:38348] connected
2020/11/12 13:21:13 [2/0/0] [client 192.168.1.80:38348] OPTIONS
2020/11/12 13:21:13 [2/0/0] [client 192.168.1.80:38348] DESCRIBE
2020/11/12 13:21:13 [2/0/0] [path cam1_low] rtsp source started
2020/11/12 13:21:13 [2/0/0] [client 192.168.1.80:38346] DESCRIBE
2020/11/12 13:21:13 [2/0/0] [path cam1_low] connecting to rtsp source
2020/11/12 13:21:13 [2/0/0] [path cam1_low] rtsp source ready
2020/11/12 13:21:13 [2/0/0] [client 192.168.1.80:38346] SETUP
2020/11/12 13:21:13 [2/0/0] [client 192.168.1.80:38348] SETUP
2020/11/12 13:21:13 [2/0/0] [client 192.168.1.80:38346] PLAY
2020/11/12 13:21:13 [2/0/1] [client 192.168.1.80:38346] is reading from path 'cam1_low', 1 track with tcp
2020/11/12 13:21:13 [2/0/1] [client 192.168.1.80:38348] PLAY
2020/11/12 13:21:13 [2/0/2] [client 192.168.1.80:38348] is reading from path 'cam1_low', 1 track with tcp
2020/11/12 13:21:14 [2/0/1] [client 192.168.1.80:38346] ERR: read tcp 192.168.1.218:8554->192.168.1.80:38346: wsarecv: An existing connection was forcibly closed by the remote host.
2020/11/12 13:21:14 [1/0/1] [client 192.168.1.80:38346] disconnected
2020/11/12 13:21:38 [1/0/1] [client 192.168.1.80:38348] TEARDOWN
2020/11/12 13:21:38 [0/0/0] [client 192.168.1.80:38348] disconnected
2020/11/12 13:21:43 [1/0/0] [client 192.168.1.80:38358] connected
2020/11/12 13:21:43 [1/0/0] [client 192.168.1.80:38358] OPTIONS
2020/11/12 13:21:43 [1/0/0] [client 192.168.1.80:38358] DESCRIBE
2020/11/12 13:21:43 [1/0/0] [client 192.168.1.80:38358] SETUP
2020/11/12 13:21:43 [1/0/0] [client 192.168.1.80:38358] PLAY
2020/11/12 13:21:43 [1/0/1] [client 192.168.1.80:38358] is reading from path 'cam1_low', 1 track with tcp
2020/11/12 13:22:13 [1/0/1] [client 192.168.1.80:38358] GET_PARAMETER
2020/11/12 13:22:43 [1/0/1] [client 192.168.1.80:38358] GET_PARAMETER
2020/11/12 13:23:13 [1/0/1] [path cam1_low] rtsp source ERR: EOF
2020/11/12 13:23:13 [0/0/0] [client 192.168.1.80:38358] disconnected
2020/11/12 13:23:13 [1/0/0] [client 192.168.1.80:38368] connected
2020/11/12 13:23:13 [1/0/0] [client 192.168.1.80:38368] OPTIONS
2020/11/12 13:23:13 [1/0/0] [client 192.168.1.80:38368] DESCRIBE
2020/11/12 13:23:13 [1/0/0] [client 192.168.1.80:38368] ERR: no one is publishing to path 'cam1_low'
2020/11/12 13:23:13 [0/0/0] [client 192.168.1.80:38368] disconnected
2020/11/12 13:23:18 [0/0/0] [path cam1_low] connecting to rtsp source
2020/11/12 13:23:18 [0/0/0] [path cam1_low] rtsp source ready
2020/11/12 13:23:18 [1/0/0] [client 192.168.1.80:38370] connected
2020/11/12 13:23:18 [1/0/0] [client 192.168.1.80:38370] OPTIONS
2020/11/12 13:23:18 [1/0/0] [client 192.168.1.80:38370] DESCRIBE
2020/11/12 13:23:18 [1/0/0] [client 192.168.1.80:38370] SETUP
2020/11/12 13:23:18 [1/0/0] [client 192.168.1.80:38370] PLAY
2020/11/12 13:23:18 [1/0/1] [client 192.168.1.80:38370] is reading from path 'cam1_low', 1 track with tcp
2020/11/12 13:23:49 [1/0/1] [client 192.168.1.80:38370] GET_PARAMETER
2020/11/12 13:24:19 [1/0/1] [client 192.168.1.80:38370] GET_PARAMETER
2020/11/12 13:24:49 [1/0/1] [client 192.168.1.80:38370] GET_PARAMETER
2020/11/12 13:25:18 [1/0/1] [path cam1_low] rtsp source ERR: EOF
2020/11/12 13:25:18 [0/0/0] [client 192.168.1.80:38370] disconnected
2020/11/12 13:25:18 [1/0/0] [client 192.168.1.80:38378] connected
2020/11/12 13:25:18 [1/0/0] [client 192.168.1.80:38378] OPTIONS
2020/11/12 13:25:18 [1/0/0] [client 192.168.1.80:38378] DESCRIBE
2020/11/12 13:25:18 [1/0/0] [client 192.168.1.80:38378] ERR: no one is publishing to path 'cam1_low'
2020/11/12 13:25:18 [0/0/0] [client 192.168.1.80:38378] disconnected
2020/11/12 13:25:23 [0/0/0] [path cam1_low] connecting to rtsp source
2020/11/12 13:25:23 [0/0/0] [path cam1_low] rtsp source ready
2020/11/12 13:25:23 [1/0/0] [client 192.168.1.80:38380] connected
2020/11/12 13:25:23 [1/0/0] [client 192.168.1.80:38380] OPTIONS
2020/11/12 13:25:23 [1/0/0] [client 192.168.1.80:38380] DESCRIBE
2020/11/12 13:25:23 [1/0/0] [client 192.168.1.80:38380] SETUP
2020/11/12 13:25:23 [1/0/0] [client 192.168.1.80:38380] PLAY
2020/11/12 13:25:23 [1/0/1] [client 192.168.1.80:38380] is reading from path 'cam1_low', 1 track with tcp
2020/11/12 13:25:26 [0/0/0] [client 192.168.1.80:38380] disconnected
2020/11/12 13:25:36 [0/0/0] [path cam1_low] rtsp source stopped
```

## Did you attach a network dump?

yes

[rtsp.zip](https://github.com/aler9/rtsp-simple-server/files/5530619/rtsp.zip)

**Comment by aler9** [comment]
Hello, it seems like the camera is forcefully closing the connection after a timeout (2 mins), so rtsp-simple-server's proxy mode is not doing its job properly - unfortunately i can't understand why, since the dump tells that rtsp-simple-server is correctly sending keepalives packets to the camera.

To fix this, i need a dump of at least 3 minutes of streaming between VLC and the camera (with TCP), so i can compare the new dump with the previous one and see why it works, and why the older one doesn't.

Thanks.

**Comment by ilfuriano** [comment]
This is the dump between VLC and Camera of 3 min.

[rtsp_3_min.zip](https://github.com/aler9/rtsp-simple-server/files/5612006/rtsp_3_min.zip)

Thanks.

**Comment by aler9** [comment]
Hello, the only difference between VLC and rtsp-simple-server is that the former sends an additional RTCP packet.
Please try this nightly release and let me know if it fixes the problem. Thanks
[rtsp-simple-server_v0.12.2-6-g4156aac_windows_amd64.zip](https://github.com/aler9/rtsp-simple-server/files/5612281/rtsp-simple-server_v0.12.2-6-g4156aac_windows_amd64.zip)
[rtsp-simple-server_v0.12.2-6-g4156aac_linux_arm64v8.tar.gz](https://github.com/aler9/rtsp-simple-server/files/5612282/rtsp-simple-server_v0.12.2-6-g4156aac_linux_arm64v8.tar.gz)
[rtsp-simple-server_v0.12.2-6-g4156aac_linux_arm7.tar.gz](https://github.com/aler9/rtsp-simple-server/files/5612283/rtsp-simple-server_v0.12.2-6-g4156aac_linux_arm7.tar.gz)
[rtsp-simple-server_v0.12.2-6-g4156aac_linux_arm6.tar.gz](https://github.com/aler9/rtsp-simple-server/files/5612284/rtsp-simple-server_v0.12.2-6-g4156aac_linux_arm6.tar.gz)
[rtsp-simple-server_v0.12.2-6-g4156aac_linux_amd64.tar.gz](https://github.com/aler9/rtsp-simple-server/files/5612285/rtsp-simple-server_v0.12.2-6-g4156aac_linux_amd64.tar.gz)
[rtsp-simple-server_v0.12.2-6-g4156aac_darwin_amd64.tar.gz](https://github.com/aler9/rtsp-simple-server/files/5612286/rtsp-simple-server_v0.12.2-6-g4156aac_darwin_amd64.tar.gz)

**Comment by ilfuriano** [comment]
There is an error and the client does not connect to the stream

``` text
2020/11/29 15:31:56 [0/0/0] rtsp-simple-server v0.12.2-6-g4156aac
2020/11/29 15:31:56 [0/0/0] [UDP/RTP server] opened on :8000
2020/11/29 15:31:56 [0/0/0] [UDP/RTCP server] opened on :8001
2020/11/29 15:31:56 [0/0/0] [TCP server] opened on :8554
2020/11/29 15:31:58 [1/0/0] [client 192.168.1.99:40717] connected
2020/11/29 15:31:58 [1/0/0] [client 192.168.1.99:40717] OPTIONS
2020/11/29 15:31:58 [1/0/0] [client 192.168.1.99:40717] DESCRIBE
2020/11/29 15:31:58 [1/0/0] [path cam1_low] rtsp source started
2020/11/29 15:31:58 [1/0/0] [path cam1_low] connecting to rtsp source
2020/11/29 15:31:58 [1/0/0] [path cam1_low] rtsp source ERR: unable to get the track clock rate: invalid format (96 H264/90000 )
2020/11/29 15:32:03 [1/0/0] [path cam1_low] connecting to rtsp source
2020/11/29 15:32:03 [1/0/0] [path cam1_low] rtsp source ERR: unable to get the track clock rate: invalid format (96 H264/90000 )
2020/11/29 15:32:08 [1/0/0] [client 192.168.1.99:40717] ERR: publisher of path 'cam1_low' has timed out
2020/11/29 15:32:08 [0/0/0] [client 192.168.1.99:40717] disconnected
2020/11/29 15:32:08 [0/0/0] [path cam1_low] connecting to rtsp source
2020/11/29 15:32:08 [0/0/0] [path cam1_low] rtsp source ERR: unable to get the track clock rate: invalid format (96 H264/90000 )
2020/11/29 15:32:13 [1/0/0] [client 192.168.1.99:40724] connected
2020/11/29 15:32:13 [1/0/0] [client 192.168.1.99:40724] OPTIONS
2020/11/29 15:32:13 [1/0/0] [client 192.168.1.99:40724] DESCRIBE
2020/11/29 15:32:13 [1/0/0] [client 192.168.1.99:40724] ERR: no one is publishing to path 'cam1_low'
2020/11/29 15:32:13 [0/0/0] [client 192.168.1.99:40724] disconnected
2020/11/29 15:32:13 [0/0/0] [path cam1_low] connecting to rtsp source
2020/11/29 15:32:13 [0/0/0] [path cam1_low] rtsp source ERR: unable to get the track clock rate: invalid format (96 H264/90000 )
2020/11/29 15:32:18 [1/0/0] [client 192.168.1.99:40726] connected
2020/11/29 15:32:18 [1/0/0] [client 192.168.1.99:40726] OPTIONS
2020/11/29 15:32:18 [1/0/0] [client 192.168.1.99:40726] DESCRIBE
2020/11/29 15:32:18 [1/0/0] [client 192.168.1.99:40726] ERR: no one is publishing to path 'cam1_low'
2020/11/29 15:32:18 [0/0/0] [client 192.168.1.99:40726] disconnected
2020/11/29 15:32:18 [0/0/0] [path cam1_low] connecting to rtsp source
2020/11/29 15:32:18 [0/0/0] [path cam1_low] rtsp source ERR: unable to get the track clock rate: invalid format (96 H264/90000 )
2020/11/29 15:32:23 [1/0/0] [client 192.168.1.99:40727] connected
2020/11/29 15:32:23 [1/0/0] [client 192.168.1.99:40727] OPTIONS
2020/11/29 15:32:23 [1/0/0] [client 192.168.1.99:40727] DESCRIBE
2020/11/29 15:32:23 [1/0/0] [client 192.168.1.99:40727] ERR: no one is publishing to path 'cam1_low'
2020/11/29 15:32:23 [0/0/0] [client 192.168.1.99:40727] disconnected
```

**Comment by aler9** [comment]
Hello, sorry for the bug but it's very difficult to debug remotely. Please try this new nightly release and let me know if it works. Thanks
[rtsp-simple-server_v0.12.2-7-g8011ff6_windows_amd64.zip](https://github.com/aler9/rtsp-simple-server/files/5612377/rtsp-simple-server_v0.12.2-7-g8011ff6_windows_amd64.zip)
[rtsp-simple-server_v0.12.2-7-g8011ff6_linux_arm64v8.tar.gz](https://github.com/aler9/rtsp-simple-server/files/5612378/rtsp-simple-server_v0.12.2-7-g8011ff6_linux_arm64v8.tar.gz)
[rtsp-simple-server_v0.12.2-7-g8011ff6_linux_arm7.tar.gz](https://github.com/aler9/rtsp-simple-server/files/5612379/rtsp-simple-server_v0.12.2-7-g8011ff6_linux_arm7.tar.gz)
[rtsp-simple-server_v0.12.2-7-g8011ff6_linux_arm6.tar.gz](https://github.com/aler9/rtsp-simple-server/files/5612380/rtsp-simple-server_v0.12.2-7-g8011ff6_linux_arm6.tar.gz)
[rtsp-simple-server_v0.12.2-7-g8011ff6_linux_amd64.tar.gz](https://github.com/aler9/rtsp-simple-server/files/5612381/rtsp-simple-server_v0.12.2-7-g8011ff6_linux_amd64.tar.gz)
[rtsp-simple-server_v0.12.2-7-g8011ff6_darwin_amd64.tar.gz](https://github.com/aler9/rtsp-simple-server/files/5612382/rtsp-simple-server_v0.12.2-7-g8011ff6_darwin_amd64.tar.gz)

**Comment by ilfuriano** [comment]
There is an error after few minutes
```
2020/11/29 16:32:31 [0/0/0] rtsp-simple-server v0.12.2-7-g8011ff6
2020/11/29 16:32:31 [0/0/0] [UDP/RTP server] opened on :8000
2020/11/29 16:32:31 [0/0/0] [UDP/RTCP server] opened on :8001
2020/11/29 16:32:31 [0/0/0] [TCP server] opened on :8554
2020/11/29 16:33:02 [1/0/0] [client 192.168.1.99:42133] connected
2020/11/29 16:33:02 [2/0/0] [client 192.168.1.99:42134] connected
2020/11/29 16:33:02 [2/0/0] [client 192.168.1.99:42133] OPTIONS
2020/11/29 16:33:02 [3/0/0] [client 192.168.1.99:42135] connected
2020/11/29 16:33:02 [3/0/0] [client 192.168.1.99:42134] OPTIONS
2020/11/29 16:33:02 [3/0/0] [client 192.168.1.99:42135] OPTIONS
2020/11/29 16:33:02 [3/0/0] [client 192.168.1.99:42135] DESCRIBE
2020/11/29 16:33:02 [3/0/0] [client 192.168.1.99:42133] DESCRIBE
2020/11/29 16:33:02 [3/0/0] [path cam1_low] rtsp source started
2020/11/29 16:33:02 [3/0/0] [path cam1_low] connecting to rtsp source
2020/11/29 16:33:02 [4/0/0] [client 192.168.1.99:42136] connected
2020/11/29 16:33:02 [4/0/0] [client 192.168.1.99:42136] OPTIONS
2020/11/29 16:33:02 [4/0/0] [client 192.168.1.99:42136] DESCRIBE
2020/11/29 16:33:02 [3/0/0] [client 192.168.1.99:42135] disconnected
2020/11/29 16:33:02 [3/0/0] [client 192.168.1.99:42134] DESCRIBE
2020/11/29 16:33:02 [2/0/0] [client 192.168.1.99:42136] disconnected
2020/11/29 16:33:02 [1/0/0] [client 192.168.1.99:42134] disconnected
2020/11/29 16:33:02 [1/0/0] [path cam1_low] rtsp source ready
2020/11/29 16:33:02 [1/0/0] [client 192.168.1.99:42133] SETUP
2020/11/29 16:33:02 [1/0/0] [client 192.168.1.99:42133] PLAY
2020/11/29 16:33:02 [1/0/1] [client 192.168.1.99:42133] is reading from path 'cam1_low', 1 track with tcp
2020/11/29 16:33:32 [1/0/1] [client 192.168.1.99:42133] GET_PARAMETER
2020/11/29 16:34:02 [1/0/1] [client 192.168.1.99:42133] GET_PARAMETER
2020/11/29 16:34:32 [1/0/1] [client 192.168.1.99:42133] GET_PARAMETER
2020/11/29 16:35:02 [1/0/1] [path cam1_low] rtsp source ERR: EOF
2020/11/29 16:35:02 [0/0/0] [client 192.168.1.99:42133] disconnected
2020/11/29 16:35:02 [1/0/0] [client 192.168.1.99:42186] connected
2020/11/29 16:35:02 [1/0/0] [client 192.168.1.99:42186] OPTIONS
2020/11/29 16:35:02 [1/0/0] [client 192.168.1.99:42186] DESCRIBE
2020/11/29 16:35:02 [1/0/0] [client 192.168.1.99:42186] ERR: no one is publishing to path 'cam1_low'
2020/11/29 16:35:02 [0/0/0] [client 192.168.1.99:42186] disconnected
2020/11/29 16:35:07 [0/0/0] [path cam1_low] connecting to rtsp source
2020/11/29 16:35:07 [1/0/0] [client 192.168.1.99:42192] connected
2020/11/29 16:35:07 [1/0/0] [client 192.168.1.99:42192] OPTIONS
2020/11/29 16:35:07 [1/0/0] [client 192.168.1.99:42192] DESCRIBE
2020/11/29 16:35:07 [1/0/0] [client 192.168.1.99:42192] ERR: no one is publishing to path 'cam1_low'
2020/11/29 16:35:07 [0/0/0] [client 192.168.1.99:42192] disconnected
2020/11/29 16:35:07 [0/0/0] [path cam1_low] rtsp source ready
2020/11/29 16:35:12 [1/0/0] [client 192.168.1.99:42193] connected
2020/11/29 16:35:12 [1/0/0] [client 192.168.1.99:42193] OPTIONS
2020/11/29 16:35:12 [1/0/0] [client 192.168.1.99:42193] DESCRIBE
2020/11/29 16:35:12 [1/0/0] [client 192.168.1.99:42193] SETUP
2020/11/29 16:35:12 [1/0/0] [client 192.168.1.99:42193] PLAY
2020/11/29 16:35:12 [1/0/1] [client 192.168.1.99:42193] is reading from path 'cam1_low', 1 track with tcp
```

**Comment by ilfuriano** [comment]
There is an error after few minutes:
```
2020/11/29 16:32:31 [0/0/0] rtsp-simple-server v0.12.2-7-g8011ff6
2020/11/29 16:32:31 [0/0/0] [UDP/RTP server] opened on :8000
2020/11/29 16:32:31 [0/0/0] [UDP/RTCP server] opened on :8001
2020/11/29 16:32:31 [0/0/0] [TCP server] opened on :8554
2020/11/29 16:33:02 [1/0/0] [client 192.168.1.99:42133] connected
2020/11/29 16:33:02 [2/0/0] [client 192.168.1.99:42134] connected
2020/11/29 16:33:02 [2/0/0] [client 192.168.1.99:42133] OPTIONS
2020/11/29 16:33:02 [3/0/0] [client 192.168.1.99:42135] connected
2020/11/29 16:33:02 [3/0/0] [client 192.168.1.99:42134] OPTIONS
2020/11/29 16:33:02 [3/0/0] [client 192.168.1.99:42135] OPTIONS
2020/11/29 16:33:02 [3/0/0] [client 192.168.1.99:42135] DESCRIBE
2020/11/29 16:33:02 [3/0/0] [client 192.168.1.99:42133] DESCRIBE
2020/11/29 16:33:02 [3/0/0] [path cam1_low] rtsp source started
2020/11/29 16:33:02 [3/0/0] [path cam1_low] connecting to rtsp source
2020/11/29 16:33:02 [4/0/0] [client 192.168.1.99:42136] connected
2020/11/29 16:33:02 [4/0/0] [client 192.168.1.99:42136] OPTIONS
2020/11/29 16:33:02 [4/0/0] [client 192.168.1.99:42136] DESCRIBE
2020/11/29 16:33:02 [3/0/0] [client 192.168.1.99:42135] disconnected
2020/11/29 16:33:02 [3/0/0] [client 192.168.1.99:42134] DESCRIBE
2020/11/29 16:33:02 [2/0/0] [client 192.168.1.99:42136] disconnected
2020/11/29 16:33:02 [1/0/0] [client 192.168.1.99:42134] disconnected
2020/11/29 16:33:02 [1/0/0] [path cam1_low] rtsp source ready
2020/11/29 16:33:02 [1/0/0] [client 192.168.1.99:42133] SETUP
2020/11/29 16:33:02 [1/0/0] [client 192.168.1.99:42133] PLAY
2020/11/29 16:33:02 [1/0/1] [client 192.168.1.99:42133] is reading from path 'cam1_low', 1 track with tcp
2020/11/29 16:33:32 [1/0/1] [client 192.168.1.99:42133] GET_PARAMETER
2020/11/29 16:34:02 [1/0/1] [client 192.168.1.99:42133] GET_PARAMETER
2020/11/29 16:34:32 [1/0/1] [client 192.168.1.99:42133] GET_PARAMETER
2020/11/29 16:35:02 [1/0/1] [path cam1_low] rtsp source ERR: EOF
2020/11/29 16:35:02 [0/0/0] [client 192.168.1.99:42133] disconnected
2020/11/29 16:35:02 [1/0/0] [client 192.168.1.99:42186] connected
2020/11/29 16:35:02 [1/0/0] [client 192.168.1.99:42186] OPTIONS
2020/11/29 16:35:02 [1/0/0] [client 192.168.1.99:42186] DESCRIBE
2020/11/29 16:35:02 [1/0/0] [client 192.168.1.99:42186] ERR: no one is publishing to path 'cam1_low'
2020/11/29 16:35:02 [0/0/0] [client 192.168.1.99:42186] disconnected
2020/11/29 16:35:07 [0/0/0] [path cam1_low] connecting to rtsp source
2020/11/29 16:35:07 [1/0/0] [client 192.168.1.99:42192] connected
2020/11/29 16:35:07 [1/0/0] [client 192.168.1.99:42192] OPTIONS
2020/11/29 16:35:07 [1/0/0] [client 192.168.1.99:42192] DESCRIBE
2020/11/29 16:35:07 [1/0/0] [client 192.168.1.99:42192] ERR: no one is publishing to path 'cam1_low'
2020/11/29 16:35:07 [0/0/0] [client 192.168.1.99:42192] disconnected
2020/11/29 16:35:07 [0/0/0] [path cam1_low] rtsp source ready
2020/11/29 16:35:12 [1/0/0] [client 192.168.1.99:42193] connected
2020/11/29 16:35:12 [1/0/0] [client 192.168.1.99:42193] OPTIONS
2020/11/29 16:35:12 [1/0/0] [client 192.168.1.99:42193] DESCRIBE
2020/11/29 16:35:12 [1/0/0] [client 192.168.1.99:42193] SETUP
2020/11/29 16:35:12 [1/0/0] [client 192.168.1.99:42193] PLAY
2020/11/29 16:35:12 [1/0/1] [client 192.168.1.99:42193] is reading from path 'cam1_low', 1 track with tcp
```

**Comment by CarlosLanderas** [comment]
I'm having the same problem. Will try to post logs ASAP

**Comment by aler9** [comment]
Hello, there's a good chance that this issue is solved by 9287e0fe982119381ea935821a1913a5131d8aee, included in v0.15.4 - i know that this issue is old, but if someone is willing to test the latest release against a camera that used to have this problem, it would be of great benefit to the community.
